### PR TITLE
Loops Ivaylo Petkov

### PR DESCRIPTION
**Pull request by IvayloPetkovBlankfactor**

### CODE REVIEW

File: Loops_IvayloPetkov/Merge two strings/Merge_Two_Strings/Program.cs
Comment (pyankov00) [inline Loops_IvayloPetkov/Merge two strings/Merge_Two_Strings/Program.cs:1]: What will happen if one of the words is with different length than the other ? 
Also save the result in a variable and the print it

File: Loops_IvayloPetkov/Check the sum/Check_the_sum/Program.cs
Comment (pyankov00) [inline Loops_IvayloPetkov/Check the sum/Check_the_sum/Program.cs:1]: What happens when you have for example the letter "f" in the word ? In this way you will have to write 30 and more  ifs for every letter? Think of a way without all of these if else-s

File: Loops_IvayloPetkov/Check if it is palindrome/Check_if_it_is_palindrome/Program.cs
Comment (pyankov00) [inline Loops_IvayloPetkov/Check if it is palindrome/Check_if_it_is_palindrome/Program.cs:1]: In this way with number 1231 the solution is wrong.

**Review by pyankov00 — CHANGES_REQUESTED** [review]
Do NOT use top level statements for more code readability. 
https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/tutorials/top-level-statements => use the old way described here.
All method's names should start with capital letter!

File: Loops_IvayloPetkov/Check the sum/Check_the_sum/Program.cs
Comment (pyankov00) [reply Loops_IvayloPetkov/Check the sum/Check_the_sum/Program.cs:1]: Taka a look to ASCII table for the correct solution

**Comment by IvayloPetkovBlankfactor** [comment]
Sorry for not uploading the "BuzzFizz" task... I was experiencing some problems with GitHub on the mac and it looks like that it wasn't uploaded. I have did the same on my PC and uploaded everything that needed to be fixed. I fixed the problems with the top level statements and used StringBuilder in two of the tasks, as for the ASCII table, I read the solution on the internet, I haven't come with this idea on my own. I just don't get it how the code works.

**Comment by IvayloPetkovBlankfactor** [comment]
Fixed the .DS_Store problem

**Comment by pyankov00** [comment]
Resolve conflicts

**Pull request closed without merging**